File: imagegroupview/src/main/java/com/loopeer/android/librarys/imagegroupview/OnImageClickListener.java
package com.loopeer.android.librarys.imagegroupview;

import com.loopeer.android.librarys.imagegroupview.model.SquareImage;

import java.util.ArrayList;

public interface OnImageClickListener {
        void onImageClick(SquareImage clickImage, ArrayList<SquareImage> squareImages, ArrayList<String> allImageInternetUrl);
    }

File: imagegroupview/src/main/java/com/loopeer/android/librarys/imagegroupview/OnImageClickListener.kt
package com.loopeer.android.librarys.imagegroupview

import android.view.View
import com.loopeer.android.librarys.imagegroupview.model.SquareImage


interface OnImageClickListener {
    fun onImageClick(clickImage: View, squareImage: SquareImage)
}